docs/common/window.ts

```diff
@@ -1,4 +1,5 @@
 const getNavigatorAgent = userAgent => {
+  // @ts-ignore
```

Comment:
Didn't `window.navigator` work here? Or did that cause other issues?

Reply:
My bad 😅  I made this patch before adding `global.d.ts` and forgot about it, thanks!